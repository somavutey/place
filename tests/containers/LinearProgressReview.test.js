import { shallow } from "enzyme";
import toJson from "enzyme-to-json";
import LinearWithValueLabel from "./../../components/containers/LinearProgressReview";
import {RecoilRoot} from "recoil" ;

it("ShortVideos", () => {
  const wrapper = shallow(<RecoilRoot><LinearWithValueLabel /></RecoilRoot>);
  expect(toJson(wrapper)).toMatchSnapshot();
});