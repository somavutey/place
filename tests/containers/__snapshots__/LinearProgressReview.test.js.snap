// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ShortVideos 1`] = `
<RecoilRoot_INTERNAL>
  <LinearWithValueLabel />
</RecoilRoot_INTERNAL>
`;
